lib/types.js

```diff
@@ -0,0 +1,135 @@
+
+	'use strict';
+	const os = require('os');
+	const path = require('path');
+	const Stream = require('stream').Stream;
+	const url = require('url');
+	const semver = require('semver');
```

Comment:
We don't need `semver` too. The properties it's needed for is just `npm init`, which is not relevant for us.